src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/CodeStyle/VisualBasicCodeStyleOptions.vb

```diff
@@ -59,10 +59,22 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeStyle
                 editorConfigName:="visual_basic_style_unused_value_assignment_preference",
                 defaultValue:=New CodeStyleOption2(Of UnusedValuePreference)(UnusedValuePreference.UnusedLocalVariable, NotificationOption2.Suggestion),
                 optionsBuilder:=s_allOptionsBuilder)
+
+        Public Shared ReadOnly IncludeParenthesesForMethodInvocations As [Option2](Of CodeStyleOption2(Of Boolean)) = CreateOption(
+            VisualBasicCodeStyleOptionGroups.Parentheses, NameOf(IncludeParenthesesForMethodInvocations),
+            defaultValue:=CodeStyleOptions2.TrueWithSilentEnforcement,
+            EditorConfigStorageLocation.ForBoolCodeStyleOption("visual_basic_style_parentheses_for_invocations")) ' RoamingProfileStorageLocation ??
+
+        Public Shared ReadOnly IncludeParenthesesForPropertyInvocations As [Option2](Of CodeStyleOption2(Of Boolean)) = CreateOption(
+            VisualBasicCodeStyleOptionGroups.Parentheses, NameOf(IncludeParenthesesForPropertyInvocations),
+            defaultValue:=CodeStyleOptions2.FalseWithSilentEnforcement,
+            EditorConfigStorageLocation.ForBoolCodeStyleOption("visual_basic_style_parentheses_for_properties")) ' RoamingProfileStorageLocation ??
```

Comment:
we're no longer doing this i believe.  @sharwell  @mavasani  for guidance on thse.